pkg/sql/insert_fast_path.go

```diff
@@ -282,6 +283,14 @@ func (n *insertFastPathNode) BatchedNext(params runParams) (bool, error) {
 			var err error
 			inputRow[col], err = typedExpr.Eval(params.EvalContext())
 			if err != nil {
+				// Intercept parse error due to ALTER COLUMN TYPE schema change.
+				if n.run.insertRun.insertCols[col].AlterColumnTypeInProgress {
+					code := pgerror.GetPGCode(err)
+					if code == "22P02" {
+						return false, errors.Wrapf(err,
+							"Column is under ALTER COLUMN TYPE schema change, this insert may not be supported until the schema change is finished (the original column must be dropped)")
```

Comment:
change to something like "column 'name' is still undergoing the ALTER COLUMN TYPE schema change..." The insert has already been rejected, why say "may"?